nx/lib/nx.ex

```diff
@@ -7194,6 +7215,7 @@ defmodule Nx do
   ## Options
 
     * `:axis` - the axis to multiply elements along. Defaults to `0`
+    * `:reverse` - if `true`, reverses the direction on the axis of accumulation. Defaults to `false`
```

Comment:
```suggestion

    * `:reverse` - whether to perform accumulation in the opposite direction. Defaults to `false`
```

Reply:
I don't like this because it doesn't lead the person to pass a boolean

Reply:
I'd say `` Defaults to `false` `` makes it clear, but your call!

Reply:
How about:

`* `:reverse` - a boolean which determines whether to perform accumulation in the opposite direction. Defaults to `false``

Reply:
Works for me :) Sidenote, we currently have `` `:keep_axis` - whether or not ...``